modules/Bio/EnsEMBL/G2P/Utils/Downloads.pm

```diff
@@ -243,9 +253,9 @@ sub write_data {
     }  
 
     $created = $gfd_panel_create_dates->{$gfd_panel_id} || 'No date';
-
+    my $comments = $comments_text->{$gfdc_id} || "No comment";
```

Comment:
It throws a warning `Use of uninitialized value $gfdc_id in hash element`.
You could change it to:
`my $comments = ($gfdc_id && $comments_text->{$gfdc_id}) ? $comments_text->{$gfdc_id} : "No comment";`